jbehave-support-core/src/main/java/org/jbehavesupport/core/internal/web/WebStepsUtils.java

```diff
@@ -0,0 +1,53 @@
+package org.jbehavesupport.core.internal.web;
+
+import lombok.RequiredArgsConstructor;
+import org.jbehavesupport.core.web.WebAction;
+import org.jbehavesupport.core.web.WebActionContext;
+import org.jbehavesupport.core.web.WebActionResolver;
+import org.jbehavesupport.core.web.WebProperty;
+import org.jbehavesupport.core.web.WebPropertyContext;
+import org.jbehavesupport.core.web.WebPropertyResolver;
+import org.jbehavesupport.core.web.WebWaitCondition;
+import org.jbehavesupport.core.web.WebWaitConditionContext;
+import org.jbehavesupport.core.web.WebWaitConditionResolver;
+import org.openqa.selenium.StaleElementReferenceException;
+import org.springframework.retry.annotation.Retryable;
+import org.springframework.stereotype.Component;
+
+import java.util.Map;
+
+@RequiredArgsConstructor
+@Component
+public class WebStepsUtils {
```

Comment:
My main concern is if we just don't hide some missing "wait condition" behind couple of "retries", with wishful thinking "this time it will work". What if not? Will we increase the number of retries?

Reply:
Well you are right, we are doing this. But from my experiance: This exception appear only after loading page or doing some action and it is kinda rare,  3 retries with 1 sec delay should provide enought time for page to become "stable". I reader quite a lot about this exception, and answer was always the same, if you catch this exception, you have to retri whole operation including finding the element. So I wanted to provide solution so our core steps can be used without ugly static waits solving issues with this exception.
Anothe aproach I found is that element locator check if element is stale and if so refresh it, but it doesn't solve case when element become stale after being found and before it being used.

Reply:
From what I read one of the "official" solutions is to just retry couple times.. If it fails even more times then I think its fine to let the test fail and let the user handle it in its own way. (so this proposed solution should be fine in that regard)